src/base/name/builder.rs

```diff
@@ -372,7 +373,7 @@ where
         Builder: FreezeBuilder,
     {
         self.end_label();
-        if self.len() + usize::from(origin.compose_len()) > 255 {
+        if self.len() + origin.compose_len() as usize > Dname::<Builder::Octets>::MAX_LENGTH  {
```

Comment:
We try to avoid the `as` operator as much as possible. Can you rewrite all of these using `usize::from`?

Reply:
done!